icechunk-python/python/icechunk/_icechunk_python.pyi

```diff
@@ -270,6 +290,12 @@ class Storage:
     def in_memory() -> Storage: ...
     @staticmethod
     def local_filesystem(path: str) -> Storage: ...
+    @staticmethod
+    def gcs(
+        bucket: str,
+        prefix: str | None,
+        credentials: GcsCredentials | None = None,
+    ) -> Storage: ...
 
```

Comment:
you'll need to add Gcs to the `AnyCredential` type, and to create `AnyGcsCredential`. This will help you  with typing in python for the `credentials.py` and `storage.py` files

Reply:
You'll need to add to those files too (and include in the __init__ file what you add)